### PR TITLE
[HttpsCertificate] Increase certificate version

### PR DESCRIPTION
**Pull request by javiercn**

### CODE REVIEW

**Comment by javiercn** [comment]
@Pilchie FYI

File: src/Tools/FirstRunCertGenerator/test/CertificateManagerTests.cs [hunk 1]
```diff
@@ -164,7 +164,7 @@ public void EnsureCreateHttpsCertificate_ReturnsExpiredCertificateIfVersionIsInc
             now = new DateTimeOffset(now.Year, now.Month, now.Day, now.Hour, now.Minute, now.Second, 0, now.Offset);
             _manager.EnsureAspNetCoreHttpsDevelopmentCertificate(now, now.AddYears(1), path: null, trust: false, subject: TestCertificateSubject);
 
-            CertificateManager.AspNetHttpsCertificateVersion = 2;
+            CertificateManager.AspNetHttpsCertificateVersion = 3;
```
Comment (jkotalik) [inline src/Tools/FirstRunCertGenerator/test/CertificateManagerTests.cs]:
```suggestion
            CertificateManager.AspNetHttpsCertificateVersion = CertificateManager.AspNetHttpsCertificateVersion + 1;
```

Comment (javiercn) [reply src/Tools/FirstRunCertGenerator/test/CertificateManagerTests.cs]: Sounds like a good change, but I'm not going to re-trigger the build for this, as it's not a significant improvement.

Comment (jkotalik) [reply src/Tools/FirstRunCertGenerator/test/CertificateManagerTests.cs]: Everytime we update the version, we are going to need to update this test. People are going to forget and have to respin CI each time. Why not fix the test now s.t. we don't hit this issue in the future.

Comment (javiercn) [reply src/Tools/FirstRunCertGenerator/test/CertificateManagerTests.cs]: We don't do this often, we can change it next time.

**Comment by anurse** [comment]
This missed the preview 3 train, right?

**Comment by javiercn** [comment]
@anurse Yes. It’s more complicated than we thought. I’ll give an update tomorrow.

**Comment by wtgodbe** [comment]
@javiercn is this for 3.1 GA or 3.1.1? It needs to get tactics approved today if we want it for 3.1 GA

**Comment by javiercn** [comment]
@wtgodbe We are not doing this, I'm working on a separate fix.

**Pull request closed without merging**